public/contact.php

```diff
@@ -62,8 +62,6 @@
         <h4 class="contact-address-text">COME OVER FOR COFFEE</h4>
         <h2 class="contact-address-heading"> 5th Floor, Ibukun House, <br> No.70 Adetokunbo Ademola Street, <br> Victoria Island, Lagos.</h2>
     </div>
-    <div class="col-lg-6 col-12 contact-address-map">
-        <img src="./assets/img/map.png" alt="map image">
-    </div>
+    <div class="col-lg-6 col-12 contact-address-map" id="map"></div>
```

Comment:
Can you update the id to something more specific like `contact-address-map`, using ordinary `map` is too generic and will not be scalable when we add more maps?